### ISSUE TITLE
12.0.5 Integrity Scan Errors

### ISSUE THREAD

**Issue by oleg-fiksel** [nextcloud/docker#230]
# Output

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- INVALID_HASH
		- cron.php
		- version.php

Raw output
==========
Array
(
    [core] => Array
        (
            [INVALID_HASH] => Array
                (
                    [cron.php] => Array
                        (
                            [expected] => ed8f1d1820eb03b333b5b47d1faf8fd572f061007cfd220b70ed2b023869cf65a3f5ba46c4744e2f1f8c34e757219333f25cbd094271a99acc0d1d16ff3c7b09
                            [current] => 82e44d5bdc49dee41715bf571ecc42d967727f1a0e425b11814d4688872fcf232d738aa40754da4203cfb65651358cd3d3af7391f77b1d28246e4a3f9e3ea285
                        )

                    [version.php] => Array
                        (
                            [expected] => ec095ceba6b2648b69138a23f8dc272f1c031ca4d49dc65e5c022b5dec687be8a8c2e20c44aa136e8d6069ca2b1f4ec06f9a4f1566fd02650e60e404b536eedc
                            [current] => c3f45c016d6e05d77007dcedb1bc5326fa1194d8d042c1b77347b971de54277452a73f4e39537a4e67b71b7b7da064079cbe4a5f3483625c59c5927cd2f3564c
                        )

                )

        )

)
```

# Docker image

* Image name: nextcloud:12.0.5-fpm
* IMAGE ID: 21d0147ad473
* https://github.com/nextcloud/docker/blob/26d58913be47abf611530c89c133475e546298ff/12.0/fpm/Dockerfile

# MD5s

```
#md5sum /usr/src/nextcloud/version.php /var/www/html/version.php
1741bf0036afa09ca0cedc71ba736851  /usr/src/nextcloud/version.php
1741bf0036afa09ca0cedc71ba736851  /var/www/html/version.php
#md5sum /usr/src/nextcloud/cron.php /var/www/html/cron.php
c32d275ee758a2909b41726ceb9bd042  /usr/src/nextcloud/cron.php
c32d275ee758a2909b41726ceb9bd042  /var/www/html/cron.php
```

**Comment by SnowMB** [comment]
Please give some more information. What did you do? Is this a fresh install? Did you upgrade? If yes, via web updater or docker?

**Comment by oleg-fiksel** [comment]
Upgrade from 12.0.4-fpm Docker container.

Here is my docker-compose.yml:

```
version: '3'

services:
  app:  
    image: nextcloud:12.0.5-fpm
    volumes:
      - www:/var/www/html:nocopy
      - data:/mnt/data:nocopy
    environment:
      - POSTGRES_HOST=db
    env_file:
      - db.env

  web:
    image: nginx
    ports:
      - 8001:80
    volumes:
      - nginx_config:/etc/nginx:ro
      - www:/var/www/html:ro
    depends_on:
      - app

volumes:
  data:
    driver_opts:
      type: "nfs"
      o: "addr=nfs-server,soft,rw,nolock"
      device: ":/mnt/nextcloud/data"
  www:
    driver_opts:
      type: "nfs"
      o: "addr=nfs-server,soft,rw,nolock"
      device: ":/mnt/nextcloud/www"
  nginx_config:
    driver_opts:
      type: "nfs"
      o: "addr=nfs-server,soft,rw,nolock"
      device: ":/mnt/nextcloud/nginx"
```

**Comment by oleg-fiksel** [comment]
Seem to be fixed in 13.0.0.
